templates/group-history.html

```diff
@@ -19,14 +19,11 @@
 <tr>
   <td class="history-group-header">{{ history.date }}</td>
   <td style="padding-left:10px">
-    {{- addentry('Hellfire Citadel', 'Mythic', 'M', num_hfc_bosses, history.hfc.mythic, history.hfc.mythic_total) -}}
-    {{- addentry('Hellfire Citadel', 'Heroic', 'H', num_hfc_bosses, history.hfc.heroic, history.hfc.heroic_total) -}}
-    {{- addentry('Hellfire Citadel', 'Normal', 'N', num_hfc_bosses, history.hfc.normal, history.hfc.normal_total) -}}
-    {{- addentry('Blackrock Foundry', 'Mythic', 'M', num_brf_bosses, history.brf.mythic, history.brf.mythic_total) -}}
-    {{- addentry('Blackrock Foundry', 'Heroic', 'H', num_brf_bosses, history.brf.heroic, history.brf.heroic_total) -}}
-    {{- addentry('Blackrock Foundry', 'Normal', 'N', num_brf_bosses, history.brf.normal, history.brf.normal_total) -}}
-    {{- addentry('Highmaul', 'Mythic', 'M', num_hm_bosses, history.hm.mythic, history.hm.mythic_total) -}}
-    {{- addentry('Highmaul', 'Heroic', 'H', num_hm_bosses, history.hm.heroic, history.hm.heroic_total) -}}
-    {{- addentry('Highmaul', 'Normal', 'N', num_hm_bosses, history.hm.normal, history.hm.normal_total) }}
+    {{- addentry('Hellfire Citadel', 'Mythic', 'M', num_nh_bosses, history.nh.mythic, history.nh.mythic_total) -}}
```

Comment:
Should these still be Hellfire Citadel/Blackrock Foundry? Same with the other file.